src/qiskit_sphinx_theme/theme/qiskit-sphinx-theme/page.html

```diff
@@ -28,19 +28,9 @@
 </label>
 
 {% if theme_announcement -%}
-<div class="custom-announcement">
-  <aside class="custom-announcement-content">
+<div class="announcement">
+  <aside class="announcement-content">
     {% block announcement %} {{ theme_announcement }} {% endblock announcement %}
-    {% if announcement_url -%}
-    <a href="{{announcement_url}}" class="reference internal">
-      {% if announcement_url_text -%}
-      {{announcement_url_text}}
-      {% else %}
-      Learn more
-      {%- endif %}
-      <svg class="icon"><use href="#svg-new-tab"></use></svg>
-    </a>
-    {%- endif %}
   </aside>
 </div>
 {%- endif %}
```

Comment:
How come you can't delete this entire block?

Reply:
Ah sorry I didn't explain this. We're reverting back to the original Furo code